notebooks/Planck_validation.py

```diff
@@ -0,0 +1,122 @@
+# some_file.py
```

Comment:
We already have quite a lot of crap on `notebooks`. Can you remove this file from the repo (but keep a copy locally, in case we need to get back to it)?

Reply:
Removed!